flexmeasures/ui/templates/base.html

```diff
@@ -22,8 +22,8 @@
     <link href="https://cdn.jsdelivr.net/npm/leaflet@1.7.1/dist/leaflet.css" rel="stylesheet" />
     <link href="https://unpkg.com/leaflet.markercluster@1.4.1/dist/MarkerCluster.css" rel="stylesheet" />
     <link href="https://unpkg.com/leaflet.markercluster@1.4.1/dist/MarkerCluster.Default.css" rel="stylesheet" />
-    <!-- Bootstrap -->
-    <link href="https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" />
+    <!-- Latest Bootstrap link-->
+    <link href="https://stackpath.bootstrapcdn.com/bootstrap/5.3.3/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
```

Comment:
Can you explain or point us to the need for this new `integrity` field?

Reply:
The integrity attribute in the <link> tag is used as a security feature to ensure that the file being fetched from an external source (in this case, the Bootstrap CSS file from the Bootstrap CDN) has not been tampered with. It's a way to verify the integrity of the file being retrieved.

Reply:
Thanks